src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -79,6 +79,11 @@ public void execute_invalidIndexFilteredList_throwsCommandException() {
         assertCommandFailure(deleteCommand, model, Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
     }
 
+    @Test
+    public void execute_didNotStoreDeletedPerson_throwsAssertion() {
+
```

Comment:
Wait why is this empty haha was it an accident or am I misunderstanding the testcase

Reply:
Woops! It was meant to be deleted... Thanks for pointing it out. :D